src/bundle/Resources/public/scss/_alloyeditor-buttons.scss

```diff
@@ -9,6 +9,9 @@
                 width: auto;
                 height: auto;
                 margin: 0.3rem 0.8rem;
+                display: flex;
+                justify-content: center;
+                align-items: center;
```

Comment:
@dew326 Icons within `.ae-button` buttons are still misaligned despite this change. When testing it I can only see an effective change if flex properties are applied in `alloyeditor-ez.css` (line 289), for instance.